src/components/calcite-tip-manager/calcite-tip-manager.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { CalciteTipManager } from "./calcite-tip-manager";
+// import { CalciteTipManager } from "./calcite-tip-manager";
```

Comment:
I turned on squigglies and started fixing some things :)

Reply:
This should be fixed or removed. #nodeadcomments

Reply:
The test is valuable but it can't be ran yet because of a Stencil bug. I can't skip the import so have to comment it or remove it. I'd prefer to keep it commented as the test will be easier to turn back on once the spec tests are working. But if you feel strongly, I'll remove.

Reply:
OK, let's keep this for now. Maybe add a comment with what you described? I know it'll stop me from commenting on the same thing in the future. 😄